### PR TITLE
docs: add lint & prettier instructions to contributing

### LINKED ISSUE #548: `style_guide`; modify Angular to our needs; probably should inporporated in `contributing.md`;

### PR DESCRIPTION
**Pull request by TobiTenno**
closes #548

### CODE REVIEW

**Comment by TobiTenno** [comment]
@tukusejssirs @emagnier would appreciate a quick review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,3 +56,25 @@ $ npm ci
 ```
 
 And you're bootstrapped. Thanks for contributing!
+
+#### Code Style
+
+We leverage typescript eslint and prettier to enforce code style. You can run the following command to check if your code is compliant:
```
Comment (tukusejssirs) [inline CONTRIBUTING.md]:
```suggestion
We leverage TypeScript Eslint and Prettier to enforce code style. You can run the following command to check if your code is compliant:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,3 +56,25 @@ $ npm ci
...
+Code style:
+
+```shell
+$ npm run lint
+
+# to fix it
+$ npm run lint:fix
+```
+
+File formatting (prettier):
```
Comment (tukusejssirs) [inline CONTRIBUTING.md]:
```suggestion
File formatting (Prettier):
```

**Review by tukusejssirs — CHANGES_REQUESTED** [review]

**Comment by tukusejssirs** [comment]
I have just some nitpicks, otherwise this is good to go.

**Comment by TobiTenno** [comment]
I don't think these make sense, given they're always referenced in code without title case. That said, I'll update it.

**Comment by tukusejssirs** [comment]
> I don't think these make sense, given they're always referenced in code without title case. That said, I'll update it.

Why don’t you think they make sense? :thinking:

IMHO any name (e.g. personal name, program name) should be spelled (incl the case) the way the owner/author wants it. Therefore, if the author of Prettier spells the program name as _Prettier_ (with a capital initial letter), we should honour it.

**Review by tukusejssirs — APPROVED** [review]

**Comment by TobiTenno** [comment]
@tukusejssirs I dismissed and would appreciate another review, as I changed how my references work to better indicate that i'm trying to reference the package names and _not_ the framework name.

As such, I titlecased Prettier where it is referenced as the framework, and code-blocked the names when referencing the scope & package names

**Review by tukusejssirs — APPROVED** [review]
Thanks, @TobiTenno! :pray:

Yeah, I have no problem with program/command names being cased as they would be run in a terminal when formatting them as code.

**Pull request merged**